fbgemm_gpu/codegen/training/backward/embedding_backward_split_kernel_warp_template.cu

```diff
@@ -658,11 +683,12 @@ hip_split_embedding{{ ndesc }}_backward_codegen_{{ optimizer }}_{{ wdesc }}{{ vd
     {%- endif %}
     const int32_t max_D,
     const int32_t max_vecs_per_thread,
-    magic_div_u32_t batch_mdiv, // PR23 extra
-    const int32_t batch, // PR23 extra
-    const int32_t num_rows, // PR23 extra
-    const int32_t num_tables, // PR23 extra
-    {{optimizer}}_kernel_arg_t opt_karg
+    {%- if is_index_select %}
```

Comment:
(You probably want to keep the `// PR23` as markers so you can locate the "hot spot" easily, but this is up to you)

Reply:
IMHO PR23 marker is confusing from the very start since you still need to be in context of what is PR23, how is it related to PR59 and what's going on in general. And I personally prefer to inspect diff to identify hotspots. However if you see that it potentially might be needed by someone else or you use it yourself, feel free to insist and I'll bring it back.